concourse/pipelines/create-cloudfoundry.yml

```diff
@@ -2109,13 +2109,18 @@ jobs:
           run:
             path: sh
             args:
-            - -e
```

Comment:
I'm uncomfortable about removing this because it's the best kind of error checking you get in shell scripting.

Without it the script will continue if any of the previous `bosh` commands fail and we may end up in an unknown state. I can't tell for sure what the `grep` will do if `vms.txt` is empty?

Can you explain why it needed to be removed? We might be able to find another solution.